package.json

```diff
@@ -14,66 +14,70 @@
     "e2e": "node test/e2e/runner.js",
     "test": "npm run unit && npm run e2e"
   },
-  "devDependencies": {
-    "babel-core": "^6.0.0",
-    "babel-loader": "^6.0.0",
-    "babel-plugin-transform-runtime": "^6.0.0",
-    "babel-preset-es2015": "^6.0.0",
-    "babel-preset-stage-2": "^6.0.0",
-    "babel-runtime": "^6.0.0",
-    "chromedriver": "^2.21.2",
-    "connect-history-api-fallback": "^1.1.0",
-    "copy-webpack-plugin": "^1.1.1",
-    "cross-env": "1.0.7",
-    "cross-spawn": "^2.1.5",
-    "css-loader": "^0.23.0",
-    "debug-menu": "^0.3.0",
-    "electron-packager": "^7.0.0",
-    "electron-prebuilt": "1.1.0",
-    "electron-rebuild": "^1.1.3",
-    "eslint": "^2.0.0",
-    "eslint-config-standard": "^5.1.0",
-    "eslint-friendly-formatter": "^1.2.2",
-    "eslint-loader": "^1.3.0",
-    "eslint-plugin-html": "^1.3.0",
-    "eslint-plugin-promise": "^1.0.8",
-    "eslint-plugin-standard": "^1.3.2",
-    "eventsource-polyfill": "^0.9.6",
-    "express": "^4.13.3",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "file-loader": "^0.8.4",
-    "html-webpack-plugin": "^2.8.1",
-    "http-proxy-middleware": "^0.11.0",
-    "inject-loader": "^2.0.1",
-    "isparta-loader": "^2.0.0",
-    "jasmine-core": "^2.4.1",
-    "json-loader": "^0.5.4",
-    "karma": "^0.13.15",
-    "karma-coverage": "^0.5.5",
-    "karma-electron-launcher": "^0.1.0",
-    "karma-jasmine": "^0.3.6",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-spec-reporter": "^0.0.24",
-    "karma-webpack": "^1.7.0",
-    "mkdirp": "^0.5.1",
-    "moment": "2.12.0",
-    "ncp": "^2.0.0",
-    "nightwatch": "^0.8.18",
-    "phantomjs-prebuilt": "^2.1.3",
-    "rimraf": "^2.5.0",
-    "selenium-server": "2.52.0",
-    "tree-kill": "^1.1.0",
-    "url-loader": "^0.5.7",
-    "vue": "^1.0.17",
-    "vue-hot-reload-api": "^1.2.0",
-    "vue-html-loader": "^1.0.0",
-    "vue-loader": "^8.3.0",
-    "vue-router": "^0.7.11",
-    "vue-style-loader": "^1.0.0",
-    "webpack": "^1.12.2",
-    "webpack-dev-middleware": "^1.4.0",
+    "devDependencies": {
+    "babel-core": "6.11.4",
+    "babel-loader": "6.2.4",
+    "babel-plugin-transform-runtime": "6.9.0",
+    "babel-preset-es2015": "6.9.0",
+    "babel-preset-stage-2": "6.11.0",
+    "babel-runtime": "6.9.2",
+    "chai": "3.5.0",
+    "chromedriver": "2.21.2",
+    "connect-history-api-fallback": "1.2.0",
+    "copy-webpack-plugin": "3.0.1",
+    "cross-env": "2.0.0",
+    "cross-spawn": "4.0.0",
+    "css-loader": "0.23.1",
+    "debug-menu": "0.4.0",
+    "electron-packager": "7.3.0",
+    "electron-prebuilt": "1.3.0",
```

Comment:
As of version 1.3.1, `electron-prebuilt` has been renamed to `electron` and the previous name will be deprecated by the end of the year.